modules/sdk-unified-wallet/src/ecdsa-unified-wallet/types.ts

```diff
@@ -1,4 +1,4 @@
 import { CoinFamily } from '@bitgo/statics';
 
-export const supportedCoins: CoinFamily[] = [CoinFamily.ETH, CoinFamily.BSC, CoinFamily.POLYGON];
+export const supportedCoins: string[] = [CoinFamily.ETH, CoinFamily.POLYGON];
```

Comment:
why did we remove BSC? not supported?

Reply:
TSS not yet supported on BSC and will lead to error when trying to create a wallet.